stylus/ui-components/button.styl

```diff
@@ -106,6 +106,11 @@ $button--upload
     background-position  em(17px, 16px) center
     background-repeat    no-repeat
 
+$button--upload-menu-item
```

Comment:
I'm wondering if a more generic name, like $button--upload-link or something, wouldn't be better. Just in case it is re-used out of any menu.